include/swift/AST/Types.h

```diff
@@ -2695,10 +2695,10 @@ enum class FunctionTypeRepresentation : uint8_t {
   /// A "thin" function that needs no context.
   Thin,
   
-  /// A C function pointer, which is thin and also uses the C calling
-  /// convention.
+  /// A C function pointer (or reference), which is thin and also uses the C
+  /// calling convention.
```

Comment:
Please also update the comment on `Uncommon::ClangFunctionType` to mention references.

Reply:
Done.